<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="radio1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1. Maintain Faculty Data</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>2. Display Student Data</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>3. Add Student to a Course</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>4. Add/ Modify a Section to a Course</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>5. Flags</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>390</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>6. Maintain Student Data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EnterButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>550</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>550</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>450</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Maintian Course Data</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
